.github/workflows/ci.yml

```diff
@@ -78,6 +78,11 @@ jobs:
         with:
           bundler-cache: true
 
+      - name: Install system deps
+        run: |
+          sudo apt-get -y update
+          sudo apt-get install -yqq libvips-dev
```

Comment:
Is there no built in or third party Action to get us this? If not, is this the recommended way for CI? Genuine question.